app/front_page/content.html

```diff
@@ -3,23 +3,23 @@
 
   <div class="col-xs-3">
   <select class="form-control" ng-model="vm.transfer.type" ng-disabled="vm.transfer.components.length > 0">
-    <option value="standard" selected>Standard</option>
-    <option value="unzipped bag">Unzipped Bag</option>
-    <option value="zipped bag">Zipped bag</option>
-    <option value="dspace">DSpace</option>
-    <option value="disk image">Disk Image</option>
+    <option value="standard" selected>{{ "Standard" | translate }}</option>
+    <option value="unzipped bag">{{ "Unzipped bag" | translate }}</option>
+    <option value="zipped bag">{{ "Zipped bag" | translate }}</option>
+    <option value="dspace">{{ "DSpace" | translate }}</option>
+    <option value="disk image">{{ "Disk image" | translate }}</option>
   </select>
-  <div class="help-block">Transfer type</div>
+  <div class="help-block">{{ "Transfer type" | translate }}</div>
   </div>
 
   <div class="col-xs-3" ng-if="vm.transfer.type !== 'zipped bag'">
   <input class="form-control" ng-model="vm.transfer.name"/>
-  <div class="help-block">Transfer name</div>
+  <div class="help-block">{{ "Transfer name" | translate }}</div>
   </div>
 
   <div class="col-xs-2">
   <input class="form-control" ng-model="vm.transfer.accession"/>
-  <div class="help-block">Accession no.</div>
+  <div class="help-block">{{ "Accession no." | translate }}</div>
```

Comment:
This section will probably have a conflict with #13 depending on which gets merged first.